plugins/lookup/vault_list.py

```diff
@@ -42,15 +42,15 @@
 - name: Perform multiple list operations with a single Vault login
   vars:
     paths:
-      - secret
+      - secret/metadata/path
       - sys/policies/acl
   ansible.builtin.debug:
     msg: "{{ lookup('community.hashi_vault.vault_list', *paths, auth_method='userpass', username=user, password=pwd) }}"
 
 - name: Perform multiple list operations with a single Vault login in a loop
   vars:
     paths:
-      - secret
+      - secret/metadata/path
```

Comment:
Needs to be 'secret/metadata' - will update.

Reply:
it looks like https://github.com/ansible-collections/community.hashi_vault/commit/857568ab7e72c9826cf56b5bb3691a4d950433d3 only updated the module and not the lookup